tools/scripts/ffi.toml

```diff
@@ -8,12 +8,27 @@
 description = "Run FFI tests"
 category = "ICU4X Development"
 dependencies = [
+    "verify-diplomat-gen",
     "test-capi",
     "test-cpp",
     "test-cortex-ffi",
     "test-nostd",
 ]
 
+[tasks.verify-diplomat-gen]
+description = "Verify that checked-in Diplomat bindings are up to date"
+category = "ICU4X Development"
+dependencies = [
+    "gen-c-diplomat",
+    "gen-cpp-diplomat",
+    "gen-js-diplomat",
+]
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+exec --fail-on-error git diff --exit-code
```

Comment:
suggestion: `git diff -- ffi` so that this can be run locally without consequences if you're not editing the ffi module?

Reply:
Ideally all the CI jobs can be run locally without much thought